app/helpers/mustache_helper.rb

```diff
@@ -143,7 +143,7 @@ def page_config
     }
   end
 
-  def channel_data
+  def collection_data
```

Comment:
Assignment Branch Condition size for collection_data is too high. [15.94/15]<br>Method has too many lines. [11/10]